docs/spec_draft.md

```diff
@@ -485,6 +486,67 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.compare
+
+### Semantics
+
+Performs element-wise comparison of `lhs` and `rhs` tensors based on
+`comparison_direction` and produces a `result` tensor. More formally,
+`result[i0, ..., iR-1]` =
+`compare_function(lhs[i0, ..., iR-1], rhs[i0, ..., iR-1])`.
+
+### Inputs
+
+| Name                   | Type                          |
+|------------------------|-------------------------------|
+| `lhs`                  | tensor of any supported type  |
+| `rhs`                  | tensor of any supported type  |
+| `comparison_direction` | type of comparison to perform |
```

Comment:
type of `comparison_direction` is an enum
I have skipped an optional attribute `comparison_type`

Reply:
All the values of `comparison_direction` need to be documented in the Semantics section. Similarly for `comparison_type` - if the spec skips it, how will the users know what it means?

Reply:
I just needed some pointers here, I will add an enum section in the spec and specify both `comparison_direction` and `comparison_type` enums.

Reply:
Thanks

Reply:
For the sake of completeness, here are the enums for `comparison_direction`:
https://github.com/openxla/stablehlo/blob/46e99f875a1b2ca7bbf17187b388904f8803e3e7/stablehlo/dialect/StablehloEnums.td#L103-L108

Here are the enums for `comparison_type`, and I suggest writing it in similar fashion to [this](https://github.com/openxla/stablehlo/pull/410/files#diff-c4355252a2f05bfb0090a2b8f6af3a2e83b4c89504fc408fe1cb52c0bd2e8f14R1832-R1836):
https://github.com/openxla/stablehlo/blob/46e99f875a1b2ca7bbf17187b388904f8803e3e7/stablehlo/dialect/StablehloEnums.td#L127-L131